File: easy/yetanothernumbergame/yetanothernumbergame.c
/*
    started at: Monday, 25. November 2013 11:12PM
*/

#include <stdio.h>

int main(int argc, char const *argv[])
{
    int iterations = 0;
    scanf("%d",&iterations);

    int input = 0;

    for (int i = 0; i < iterations; ++i)
    {   
        scanf("%d",&input);
        if (input % 2 == 0)
        {
            printf("ALICE\n");
        }
        else
        {
            printf("BOB\n");
        }
    }

    return 0;
}

File: easy/yetanothernumbergame/yetanothernumbergame.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# started at: Monday, 25. November 2013 11:12PM
import sys

def main():
    iterations = int(raw_input())

    for line in xrange(iterations):
        N = int(sys.stdin.readline())

        if N % 2 == 0:
            print 'ALICE'
        else:
            print 'BOB'

if __name__ == '__main__':
    main()